docs/configuration.asciidoc

```diff
@@ -217,6 +217,31 @@ It also expects that a period (`.`) is used to separate the integer and the frac
 | 1.0     | Double 
 |============
 
+[float]
+[[config-transaction-max-spans]]
+==== `TransactionMaxSpans` (performance)
```

Comment:
@SergeyKleyman, it what version such option will be available? It would be nice to mention it here.

Reply:
It's hard to say for sure but most likely the next release will be for [the milestone currently marked as 7.5 (which means .NET agent release aligned with 7.5 Elastic stack release)](https://github.com/elastic/apm-agent-dotnet/milestone/5) and agent's version will be 1.2